Makefile

```diff
@@ -24,6 +24,11 @@ IMAGE_TAG := devel
 endif
 export IMAGE_TAG
 
+CATALOGD_BINARIES=catalogd
+LINUX_BINARIES=$(join $(addprefix linux/,$(CATALOGD_BINARIES)), )
+
+BUILDCMD = go build -tags '$(GO_BUILD_TAGS)' -ldflags '$(GO_BUILD_LDFLAGS)' -gcflags '$(GO_BUILD_GCFLAGS)' -asmflags '$(GO_BUILD_ASMFLAGS)' -o $(BUILDBIN)/$(notdir $@) ./cmd/$(notdir $@)
+
```

Comment:
This shouldn't be necessary, as catalogd and operator-controller binaries should be able to use the same set of build rules/recipes.

Reply:
those have been cleaned up.